packages/ripple-keypairs/src/signing-schemes/secp256k1/index.ts

```diff
@@ -0,0 +1,64 @@
+import { numberToBytesBE } from '@noble/curves/abstract/utils'
+import { secp256k1 as nobleSecp256k1 } from '@noble/curves/secp256k1'
+import { bytesToHex } from '@xrplf/isomorphic/utils'
+
+import type {
+  DeriveKeyPairOptions,
+  HexString,
+  SigningScheme,
+} from '../../types'
+
+import { derivePrivateKey } from './utils'
+import assert from '../../utils/assert'
+import Sha512 from '../../utils/Sha512'
+
+const SECP256K1_PREFIX = '00'
+
+const secp256k1: SigningScheme = {
+  deriveKeypair(
+    entropy: Uint8Array,
+    options?: DeriveKeyPairOptions,
+  ): {
+    privateKey: string
+    publicKey: string
+  } {
+    const derived = derivePrivateKey(entropy, options)
+    const privateKey =
```

Comment:
There's a `.toUpperCase()` in the original code - does that automatically happen in the new lbirary?